Makefile

```diff
@@ -132,6 +132,10 @@ verify-genproto:
 verify-goimport:
 	PASSES="goimport" ./scripts/test.sh
 
+.PHONY: verify-yamllint
+verify-yamllint:
```

Comment:
Can you add a matching fix-yamllint rule?

Reply:
Note for context `yamllint` is only a linter, it doesn't have automatic formatting.  We would need to add another utility like `yamlfmt` for auto formatting https://github.com/google/yamlfmt.

Reply:
If they are compatible then we should. As we add linters it can get more and more annoying for contributors to fix any small thing. Here comes the `make fix` that should make addressing majority of errors reported by linters.

Reply:
Yes, I plan to add rules for `fix-yamlint`. There are many tools that can automatically format YAML, but not all of them work well.  https://github.com/adrienverge/yamllint/issues/62

I want to add it in the follow-up PR after comparing these tools. I will also try [the tool ](https://github.com/google/yamlfmt) mentioned by @jmhbjz.